UI/public/css/main.css

```diff
@@ -0,0 +1,250 @@
+  /* Nav and Footer Styling */
+
+  * {
+  	box-sizing: border-box;
+  }
+
+  body {
+  	font-family: 'Monserrat', sans-serif;
```

Comment:
These are not indented properly, also the braces should align with each other. Additionally, I thought we discussed the introduction of google font.

Reply:
I used google font on the form.css and request.css files

Reply:
I have added the cdn and deleted the fonts folder...